infer/lib/infer_pack/models.py

```diff
@@ -1156,7 +1156,10 @@ def forward(self, x):
         b, c, t = x.shape
         if t % self.period != 0:  # pad first
             n_pad = self.period - (t % self.period)
-            x = F.pad(x, (0, n_pad), "reflect")
+            if hasattr(torch, "xpu") and torch.xpu.is_available() and x.dtype == torch.bfloat16:
```

Comment:
Same. can use `if config.has_xpu: ...`

Reply:
models.py doesn't import config so i added has_xpu variable below the main imports.

Reply:
OK.